src/network/peers.rs

```diff
@@ -275,14 +281,23 @@ impl<N: Network, E: Environment> Peers<N, E> {
                 for peer_ip in self
```

Comment:
Unrelated to this line but had nowhere else to make the comment: we should consider refactoring these blocks before they get any bigger.